src/App.js

```diff
@@ -1,77 +1,26 @@
-        <p className="role-details"> Facilitated the daily operating tasks of new and
-           existing business in the DFW office by training/overseeing
-           new operations professionals. Secured fair freight rates by
-            researching the freight market daily, assessing company project
-             history, considering carrier reliability and legal transit time,
-              etc. while building/improving customer relationship.
-        </p>
-      </div>
-      <div className="container-item">
-        <h4>Logistics Coordinator</h4>
-        <p className="company">Fluor Corporation, Houston</p>
-        <p className="role-details"> Organized progress meetings with package execution teams
-           and vendors in order to attain vendor updates and problem
-          solve to guarantee timely material readiness for shipment
-          and minimize schedule impact.
-        </p>
-      </div>
-    </div>
-
-    <div className='container-item container-side'>
+import React, { Component } from 'react'
+import ReactDOM from 'react-dom'
+import Profile from './components/Profile'
+import ExperienceList from './components/ExperienceList'
+import ExperienceCard from './components/ExperienceCard'
+import Sidebar from './components/Sidebar'
+
+class App extends Component {
+  constructor() {
+    super()
+  }
+
+  render() {
+    return (
       <div>
-        <img src="../profile.jpg" alt="Profile" />
-        <h1>Dylan Thorwaldson</h1>
-        <h5>Web Development Apprentice</h5>
-        <ul className="contact">
-          <li><i className="far fa-envelope fa-sm"></i><a href="mailto:notreal@gmail.com?subject=Hey-Dylan"> dylan@gmail.com</a></li>
-          <li><i className="fas fa-phone fa-sm"></i><a href="#">  (832) 741-5282</a></li>
-          <li><i className="fab fa-chrome fa-sm"></i><a href="http://www.dylanthorwaldson.com" target="_blank" rel="noopener noreferrer">  dylanthorwaldson.com</a></li>
-          <li><i className="fab fa-linkedin-in fa-sm"></i><a href="http://www.linkedin.com/in/dtthor" target="_blank" rel="noopener noreferrer">  linkedin.com/in/dtthor</a></li>
-          <li><i className="fab fa-github fa-sm"></i><a href="http://www.github.com/dtthor" target="_blank" rel="noopener noreferrer">  www.github.com/dtthor</a></li>
-          <li><i className="fab fa-twitter fa-sm"></i><a href="http://www.twitter.com/dtthor" target="_blank" rel="noopener noreferrer">  @dtthor</a></li>
-        </ul>
-
-        <div className="education">
-          <h5>EDUCATION</h5>
-          <h6>BA in Public Relations</h6>
-          <h6 className="school">Pepperdine University</h6>
-          <h6 className="school">2008-2012</h6>
-
-        </div>
-
+        <h3 className="container-item"><i className="icon fas fa-user-circle fa-lg"></i> CAREER PROFILE</h3>
+        <Profile />
+        <h3 className="container-item"><i className="icon fas fa-briefcase fa-lg"></i> EXPERIENCE</h3>
```

Comment:
this `h3` it's repeating a log maybe make a component that takes the title (career profile), and the icon (brief case, circle)) as props?

Reply:
good idea, just made this change.